omod/src/main/resources/config.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<module configVersion="1.2">
+
+	<id>${project.parent.artifactId}</id>
+	<name>${project.parent.name}</name>
+	<version>${project.parent.version}</version>
+	<package>${project.parent.groupId}.${project.parent.artifactId}</package>
+	<author>PIH</author>
+	<description>
+		${project.parent.description}
+	</description>
```

Comment:
All modules must define a "config.xml" that define the name, version, etc... note most of these just use varialbes to reference names specified in the pom.